hack/flaky_tests_analysis/README.md

```diff
@@ -0,0 +1,84 @@
+# Flaky Tests Analysis
+
+## Backend Tests Analysis
+
+###  Setup
+
+`mix test` along with some config that makes the test runner process write to a JUnit file for every run was set up. 
+
+
+### Execution 
+
+```
+❯ make N-TIMES=500 gen-test-data
```

Comment:
This is outdated, right?
I mean now we have options in the makefile to run frontend, backend and e2e tests as far as I see,

Maybe this whole file content should be generilzed, as the usage is almost the same in all 3 forms of tests